stores/migrations.go

```diff
@@ -339,3 +354,16 @@ func performMigration00003_healthcache(txn *gorm.DB, logger glogger.Interface) e
 	logger.Info(context.Background(), "migration 00003_healthcheck complete")
 	return nil
 }
+
+func performMigration00004_objectID_collation(txn *gorm.DB, logger glogger.Interface) error {
+	if isSQLite(txn) {
```

Comment:
`nit` maybe we should only wrap the `Exec` inside the `if` statement so we have the logging in all instances, it's more correct to log the migration happened I think even in case of sqlite